File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/C/Exercicio 3.c
/* 
3 - Ler dois números, armazenando-os nas variáveis num1 e num2. Verificar se o valor de
num1 é maior que o valor de num2 e, em caso positivo, trocar os conteúdos das variáveis. 
*/

#include<stdio.h>

int main ()
{
    float num1, num2, aux;

    printf ("Digite o valor de num1: ");
    scanf ("%f",&num1);
    printf ("Digite o valor de num2: ");
    scanf ("%f",&num2);

    if (num1>num2) {
        aux=num2;
        num2=num1;
        num1=aux;
    }
    printf ("\n O valor de num2 eh: %.2f e num1 eh: %.2f\n\n", num2, num1);
    return 0;
}

File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/Dart/exercicio3.dart
// 3 - Ler dois números, armazenando-os nas variáveis num1 e num2. Verificar se o valor de
// num1 é maior que o valor de num2 e, em caso positivo, trocar os conteúdos das variáveis.

import 'dart:io';

main(List<String> args) {
  print('Digite valor de num1: ');
  int num1 = int.parse(stdin.readLineSync()!);
  print('Digite valor de num2: ');
  var num2 = int.parse(stdin.readLineSync()!);

  if (num1 > num2) {
    int aux;
    aux = num1;
    num1 = num2;
    num2 = aux;
  }

  print('Valor final de num1 é: ${num1}');
  print('Valor final de num2 é: ${num2}');
}
